CMakeLists.txt

```diff
@@ -650,6 +650,10 @@ else()
 endif()
 
 find_package(OpenAL)
+if(OPENAL_FOUND)
```

Comment:
The other definitions usually print "using from Externals" if _no_ system library was found. Did you mean `NOT OPENAL_FOUND`, or do you mean to _always_ use the one from Externals?